.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+daysUntilStale: 90
+daysUntilClose: 90
+
+exemptLabels:
+- type/bug
+- type/cleanup
+- type/feature
+- type/infrastructure
+- critical
+- security
+exemptMilestones: true
+exemptAssignees: true
+
+staleLabel: wontfix
+
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```

Comment:
How about this?

> We are closing this issue as it seems to have grown stale. If you still encounter this problem with the latest version, please feel free to re-open it.
>
> Please understand that outdated issues are closed regularly to keep the amount of open issues to a manageable minimum. In any case, thanks for taking an interest in this software and contributing by opening the issue in the first place!

Reply:
@luceos already had a template at https://github.com/flarum/core/pull/1841#issuecomment-527232258, not sure if you saw it or prefer this one instead.

We also need a comment for when the bot closes an issue.

Reply:
Ah, did not see @luceos' comment. His seems to be a combination of the existing `markComment` and my proposal. And mine doesn't fit the `markComment`, but rather the one for closing.

So I'd propose the following combination of all three...

**Marking as stale**

> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. We do this to keep the amount of open issues to a manageable minimum.
>
> In any case, thanks for taking an interest in this software and contributing by opening the issue in the first place!

**Closing because stale**

> We are closing this issue as it seems to have grown stale. If you still encounter this problem with the latest version, please feel free to re-open it.